core/commands/commit.py

```diff
@@ -71,11 +99,14 @@ def run(self, edit):
             initial_text = COMMIT_HELP_TEXT
 
         if sublime.load_settings("GitSavvy.sublime-settings").get("show_commit_diff"):
-            if option_amend:
+            if self.amending():
                 initial_text += self.git("diff", "HEAD^")
             else:
                 initial_text += self.git("diff", "--cached")
 
+        if sublime.load_settings("GitSavvy.sublime-settings").get("show_commit_summary"):
+            initial_text += self.fetch_commit_summary()
```

Comment:
Is there some reason we can't wrap it in a method vs passing it everywhere?  `amending` really does seem to be a property of `GsCommitInitializeViewCommand to me`.